osidb/tests/test_flaw.py

```diff
@@ -733,6 +740,58 @@ def test_validate_affect_ps_module_errors(self, bz_id, ps_module, should_raise):
             affect = AffectFactory(flaw=flaw, ps_module=ps_module)
             assert affect
 
+    @pytest.mark.parametrize(
+        "group,unembargo_date,should_raise",
+        [
+            (settings.PUBLIC_READ_GROUPS, None, True),
```

Comment:
Observe that `PUBLIC_READ_GROUPS` is an array while `EMBARGO_READ_GROUP` is string so you need to deal with them differently. Those ACLs are tricky :smile: and they work with many things without throwing any error but here one or the other case must be something else than intended.